magenta/interfaces/midi/README.md

```diff
@@ -66,8 +66,8 @@ install one by doing the following:
 
 **Ubuntu:** Use the command `sudo apt-get install fluid-soundfont-gm`.<br />
 **Mac:** Download the soundfont from
-http://www.musescore.org/download/fluid-soundfont.tar.gz and unpack the SF2
-file.
+ftp://ftp.osuosl.org/pub/musescore/soundfont/fluid-soundfont.tar.gz and unpack the SF2
```

Comment:
Can you change this to an http link? That seems to work, and http is more widely supported by browsers.